membership-attribute-service/app/services/ScanamoAttributeService.scala

```diff
@@ -73,4 +75,25 @@ class ScanamoAttributeService(client: AmazonDynamoDBAsync, table: String)
   }
 
   override def delete(userId: String): Future[DeleteItemResult] = run(scanamo.delete('UserId -> userId))
+
+  override def ttlAgeCheck(dynamoAttributes: Option[Attributes], identityId: String, currentDate: LocalDate): Unit = {
+    // Dynamo typically cleans up within 48 hours of expiry. I'm allowing slightly older expiry dates to avoid false alarms
+    // https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/howitworks-ttl.html
+    val oldestAcceptedAge = currentDate.toDateTimeAtCurrentTime.minusDays(3)
+    val tooOld = for {
+      attributes <- dynamoAttributes
+      ttl <- attributes.TTLTimestamp
```

Comment:
after a while we can change it to if the timestamp is None or if it's too old

Reply:
We still have an outstanding task to add a TTL to all items in the table (will probably wait a few days and see how much happens naturally first). Once this happens I think we'd like to remove the optionality for TTL.